### PR TITLE
Autotool-3581 data group fails when cli preference list all properties is enabled

### PR DESCRIPTION
**Pull request by seattlevine**

### CODE REVIEW

File: test/storageDataGroup.js [hunk 1]
```diff
@@ -45,6 +45,8 @@ describe('StorageDataGroup', () => {
             '        }',
             '    }',
             '    partition appsvcs',
+            '    description this is my group',
+            '    app-service my app service',
```
Comment (ansmithf5) [inline test/storageDataGroup.js:49]: I agree I think just changing this specific piece is good, instead of trying to make something more generic.

**Review by ansmithf5 — APPROVED** [review]
Looks good to me, good call on just adding a catch.

**Review by RobCupples — APPROVED** [review]
Looks ok to me.

**Review by mstokes-f5 — APPROVED** [review]
Looks good to me!

**Pull request merged**